### PR TITLE
chore: add editor repo to init command

### PR DESCRIPTION
**Pull request by leanmendoza**
# What?
- Add --project param to choose between `scene` and `editor`
- Add back from previous CLI the param --template (internal)

With editor PR closes https://github.com/decentraland/sdk/issues/749

### CODE REVIEW

File: packages/@dcl/sdk-commands/src/commands/init/index.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ export const args = declareArgs({
   '--yes': Boolean,
   '-y': '--yes',
   '--dir': String,
-  '--skip-install': Boolean
+  '--skip-install': Boolean,
+  '--scene': String
```
[the remark this answers is not in the document]
Comment (menduz) [reply packages/@dcl/sdk-commands/src/commands/init/index.ts:22]: There will be possible to create other things than scenes soon, like wearables

[the remark this answers is not in the document]
Comment (menduz) [reply packages/@dcl/sdk-commands/src/commands/init/index.ts:22]: also --template should also accept .zip urls like the old cli

[the remark this answers is not in the document]
Comment (leanmendoza) [reply packages/@dcl/sdk-commands/src/commands/init/index.ts]: I changed it to `--project` , and add `--template` for custom URLs (for now without documentation)

[the remark this answers is not in the document]
Comment (menduz) [reply packages/@dcl/sdk-commands/src/commands/init/index.ts]: --project is already used to specify paths. `--template string|URL` doesn't work for you?

**Review by gonpombo8 — APPROVED** [review]

**Pull request merged**